pkg/api/v1alpha1/cloudstackdatacenterconfig.go

```diff
@@ -18,13 +18,9 @@ func NewCloudStackDatacenterConfigGenerate(clusterName string) *CloudStackDatace
 		},
 		Spec: CloudStackDatacenterConfigSpec{
 			Domain: "domain1",
-			Zone: CloudStackResourceRef{
-				Value: "zone1",
-				Type:  "name",
-			},
-			Network: CloudStackResourceRef{
-				Value: "net1",
-				Type:  "name",
+			Zones: []Zone{
```

Comment:
Following up on our conversation earlier, how will multi-zone interact with WorkerNodeGroup in terms of worker node placement?